src/main/java/com/javadiscord/javabot/service/help/HelpChannelManager.java

```diff
@@ -40,6 +43,14 @@ public HelpChannelManager(HelpConfig config) {
 		this.logChannel = Bot.config.get(config.getGuild()).getModeration().getLogChannel();
 	}
 
+	public static MessageEmbed getHelpChannelEmbed(String message, @Nullable Color color) {
+		return new EmbedBuilder()
+				.setDescription(message)
+				.setColor(color)
+				.setTimestamp(Instant.now())
+				.build();
+	}
```

Comment:
I don't think that this static method is necessary. Or, it would be if it offered more than a thin wrapper around something which is already easily doable in JDA. If we want to standardize how message embeds are sent, then don't allow callers to specify a custom color, but instead provide some sort of generic "type" for the message which would dictate how the internal implementation renders it.